docs/development/translations.rst

```diff
@@ -0,0 +1,449 @@
+Translations
+============
+
+Get Started
+-----------
+
+- :ref:`What is SecureDrop? <what_is_securedrop>`
+- :ref:`Who uses SecureDrop? <who_uses_securedrop>`
+- :ref:`How is SecureDrop translated? <how_is_securedrop_translated>`
+
+.. _get_started_using_weblate:
+
+Get started using Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can choose to register with your email address, or by linking a GitHub account. If you already have a GitHub account to which you’re usually logged in, that can be easier, but it’s not required.
+
+- :ref:`Register an account on Weblate using an email address <how_to_register_on_weblate_email>`
+- :ref:`Register an account on Weblate using a GitHub account <how_to_register_on_weblate_github>`
+
+You can contribute to any language, but Weblate has some conveniences to make it easier to work with your preferred languages.
+
+- :ref:`Choose your preferred languages on Weblate <how_to_manage_your_preferred_languages_weblate>`
+- :ref:`Can new languages be added to Securedrop? <adding_a_new_language>`
+
+Our Weblate instance only contains one project, SecureDrop, which has two translation components: the main SecureDrop web application (labeled "SecureDrop"), and the translations for the desktop icons of the admin and journalist workstations used by news organizations (labeled "desktop").
+
+|Weblate project page screenshot|
+
+- :ref:`Start translating a language on Weblate <how_to_translate_a_language_weblate>`
+- :ref:`How to translate a phrase on Weblate <how_to_translate_a_phrase_weblate>`
+- :ref:`Learn more about translating phrases with placeholders <translating_phrases_with_placeholders>`
+- :ref:`Learn more about translating phrases with HTML code <translating_phrases_with_html>`
+
+Once a string is translated, it must be approved by a :ref:`reviewer <weblate_glossary_reviewer>` before being accepted into SecureDrop.
+
+- :ref:`Why do strings need to be reviewed? <why_are_strings_reviewed>`
+- :ref:`How to approve a translated string <how_to_approve_a_translated_string>`
+- :ref:`How can I become a reviewer? <how_to_become_a_reviewer>`
+
+Approved strings can only be modified by :ref:`reviewers <weblate_glossary_reviewer>`. However, translators can still suggest modifications and make comments if they notice something wrong.
+SecureDrop translations are a collaborative endeavour!
+
+- :ref:`Suggesting changes to source strings <suggesting_changes_to_source_strings>`
+- :ref:`Changing an existing translation <changing_an_existing_translation>`
+
+.. _getting_help_translations:
+
+Getting help about translations
```

Comment:
I stumbled over the 'about,' I understand the nuances with the process, tool weblate, .po exports, the source strings, or a bug with the login.. but that's not relevant here. It's an introductory text for both experienced translators and just community members/users, who want to help, fix a string or expand. They probably don't care about Weblate itself. 'Getting help' can be enough here, 'Getting help with translations' or 'while translating' are, in my ears, a bit more welcoming and less abstract and dry.

Reply:
Oh, yes. I agree totally about the goals: this is not about Weblate but about getting help in the process of translating. I am not a native English speaker; in retrospect using "with translations" would probably have been closer to what I had in mind, but now that you point it out, I think that you're right and "Getting help" is clear and unambiguous.

```suggestion
Getting help
```

Reply:
Done.